KEEP/services/hiawatha

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+if ! grep -q '^www-data:' /etc/passwd ; then
+	busybox addgroup -S www-data
+	busybox adduser -h "$SRVDIR" -s /bin/nologin -S -D -H -G www-data www-data
+	if [ ! -d "$SRVDIR" ] ; then
```

Comment:
this block should be outside the passwd check since its possible that the user, but not the srvdir exists